lib/game.rb

```diff
@@ -120,42 +124,22 @@ def switch_player
         @computer_player
     end
 
-    def win?
-      @players.each do |player|
-        if is_winner?(player)
-          return true
+    def declare_outcome(board, markers)
```

Comment:
Do you mean that `Game` should call a `GameState` method just once and `GameState` will figure out whether the game is over (and if so, whether it's a draw or win)? If so, I think that's a pretty good idea.